lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -225,16 +226,21 @@ export default function SubmitGradeForm({ student }) {
             setSubmitGradeError('');
           }}
           cancelLabel="Close"
+          onRetry={e => {
+            setSubmitGradeError('');
+            onSubmitGrade(e);
+          }}
         />
       )}
       {!!fetchGradeError && (
         <ErrorDialog
           title="Fetch Grade Error"
-          error={{ message: submitGradeError }}
+          error={{ message: fetchGradeError }}
```

Comment:
This was actually a bug that I caused! It's why "Malformed LTI..." message was not reporting on fetch errors and we were not able to distinguish between any type of fetch error.